stetho/src/main/java/com/facebook/stetho/inspector/network/DecompressionHelper.java

```diff
@@ -20,8 +20,8 @@
 
 // @VisibleForTest
 public class DecompressionHelper {
```

Comment:
Given how we are doing this in multiple places now it may make sense to create an interface to allow people to specify additional compression/decompression methods in a more generic way.

For example if someone wants to use bzip they would have to re-implement `NetworkEventReporterImpl` as well as the `RequestBodyHelper`. If we had a simple registry system from Content-Encoding to an interface that exposes a 
compress and decompress function it would enable this.